### ISSUE TITLE
roslaunch problem

### ISSUE THREAD

**Issue by miguelrasteiro** [uzh-rpg/rpg_svo#60]
Hi, 

When i do:
roslaunch svo_ros test_rig3.launch

this error occurs:
[test_rig3.launch] is neither a launch file in package [svo_ros] nor is [svo_ros] a launch file name

looks like svo_ros is not added to the roslaunch command, any idea why, or how can i correct the problem?

Tks

**Comment by AmandaSutherland** [comment]
I have encountered the same problem. Does it matter where in your file path you are when you do your suggested fix? Should you be doing something different if you are working within a folder in your catkin_ws? This was the error I received. 

bash: /svo_catkin_ws/devel/setup.bash: No such file or directory

Thanks in advance.

**Comment by miguelrasteiro** [comment]
Yes, if you did install catkin correctly, you have to type the path from your current location or the full path from root.

**Comment by navneet894** [comment]
when I run the command "source /svo_catkin_ws/devel/setup.bash" iam getting "permission denied".
and When i do:
roslaunch svo_ros test_rig3.launch

this error occurs:
[test_rig3.launch] is neither a launch file in package [svo_ros] nor is [svo_ros] a launch file name


please help!!!!

**Comment by miguelrasteiro** [comment]
try "sudo source /svo_catkin_ws/devel/setup.bash" ?!

**Comment by Saums** [comment]
@miguelrasteiro Facing the same error, followed your suggestion but the error persists. 

where is svo_ros supposed to be saved?

**Comment by pinky77mm** [comment]
Even if I ran source /catkin_ws/devel/setup.bash, and the same error occurred after I ran launch command. What else can I do?

**Comment by amatii** [comment]
I have the same problem with kinetic. it happened to every other packages as well. source... did not solve the problem

**Comment by nrocketmann** [comment]
Yeah, same error here. I did the `source devel/setup.bash` thing a million times and same error still there. Any ideas/fixes?

**Comment by nimishamehta5** [comment]
> Yeah, same error here. I did the `source devel/setup.bash` thing a million times and same error still there. Any ideas/fixes?

Hey, any fixes for this? I have the same problem, but with rviz. Unable to launch rviz.

**Comment by pinky77mm** [comment]
> > Yeah, same error here. I did the `source devel/setup.bash` thing a million times and same error still there. Any ideas/fixes?
> 
> Hey, any fixes for this? I have the same problem, but with rviz. Unable to launch rviz.

Maybe you can try source at different path you have used.
